Example/WalletApp/Common/Extensions/UIKit/UIViewController.swift

```diff
@@ -0,0 +1,59 @@
+import UIKit
+import SwiftUI
+import StoreKit
+
+extension UIViewController {
```

Comment:
We need to share common things between our apps with SPM Package. Could be a good improvements may be in another PR's